README.md

```diff
@@ -1 +1,205 @@
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Leaderboard] <a name="about-project"></a>
+
+**[Leaderboard]** is a website displays scores submitted by different players. It also allows you to submit your score. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- **HTML5**
+- **CSS3**
+- **JavaScript(ES6)**
+- **Webpack**
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Add Task]**
+- **[Remove Task]**
+- **[Edit Task]**
+- **[Display Tasks]**
+- **[Remove All Tasks Checked As Completed]**
+- **[Remove A Selected Task]**
+- **[Using HTML-Webpack-Plugin]**
+- **[Using Webpack CSS-Loader]**
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://zohra-neda.github.io/Leaderboard/dist/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+  cd my-folder
+  https://github.com/Zohra-Neda/Leaderboard.git
+
+### Install
+
+Install this project with:
+
+
+Example command:
+
+  cd my-project
+  npm install
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+Example command:
+npm start
+
+### Deployment
+
+You can deploy this project using:
+
+github pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author**
+
+- GitHub: [@zohra-neda](https://github.com/zohra-neda)
+- Twitter: [@zohra_neda](https://twitter.com/zohra_neda)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/zohra-neda-3716b720b/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Add more dynamic pages]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project give ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for initiating the idea behind the project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./License) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- Hey, the license is currently broken. please, fix this to improve your readme.